docs/case-studies/gitops.md

```diff
@@ -0,0 +1,143 @@
+---
+title:  GitOps with KubeVela
+---
+
+GitOps is a continuous delivery method that allows developers to automatically deploy applications by changing code in a Git repository. In this section, you will learn how to use KubeVela for GitOps deployment.
+
+## Preparation
+
+First, prepare a Git Repository with `Application` files that need to be applied, please refer to [KubeVela-GitOps-Demo](https://github.com/FogDong/KubeVela-GitOps-Demo):
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: first-vela-workflow
+  namespace: default
+spec:
+  components:
+  - name: express-server
+    type: webservice
+    properties:
+      image: stefanprodan/podinfo
+      port: 9898
+    traits:
+    - type: ingress
+      properties:
+        domain: testsvc.example.com
+        http:
+          /: 9898
+  - name: nginx-server
+    type: webservice
+    properties:
+      image: nginx:1.21
+      port: 80
+```
+
+Second, apply a secret with your Git username and password in cluster:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-secret
+type: kubernetes.io/basic-auth
+stringData:
+  username: your username
+  password: your password
+```
+
+## Write the Application that can apply automatically
+
+Apply the `Application` file to the cluster:
```

Comment:
We have two applications here, one is from the git repo above, and the one as below. Maybe specifying the `application file` a little bit more will help:)